src/validation.ts

```diff
@@ -207,11 +211,14 @@ export function validateSettings(settings: Settings): string | undefined {
   if (!settings[checkT]) {
     return 'Settings values required';
   }
-
+  // WHAT IS THIS FUNCTION FOR??
```

Comment:
VALIDATING 😆

Reply:
As you can see, there's a lot of things we _don't_ validate on the frontend (but we probably _should_)